specification/glossary.md

```diff
@@ -161,3 +164,25 @@ Pseudorandomly resolve flag values using a context property, such as a targeting
 ### Rule
 
 A rule is some criteria that's used to determine which variant a particular context should be mapped to.
+
+## SDK Paradigms
```

Comment:
FWIW, I really prefer referring to these as two *paradigms*, rather than "modes"/